pages/modules/[module].tsx

```diff
@@ -347,65 +364,109 @@ const ModulePage: NextPage<ModulePageProps> = ({
                 </div>
               </div>
               <div id="metadata" className="sm:pl-2 basis-8 md:basis-[12rem]">
-                <h2 className="text-2xl font-bold mt-4 mb-2">Metadata</h2>
+                <h2 className="text-2xl font-bold mt-4 mb-2">About</h2>
                 <div>
                   {repoDescription && (
-                    <div className="mb-3">
-                      <p className="text-sm">{repoDescription}</p>
+                    <div className="mb-2">
+                      <p className="text-md">{repoDescription}</p>
                     </div>
                   )}
-                  {metadata.homepage !== githubLink ? (
-                    <a
-                      href={metadata.homepage}
-                      className="text-link-color hover:text-link-color-hover"
-                      title={metadata.homepage}
-                    >
-                      <FontAwesomeIcon icon={faGlobe} className="mr-1" />
-                      Homepage
-                    </a>
-                  ) : null}
-                </div>
-                {githubLink && (
-                  <div>
-                    <a
-                      href={githubLink}
-                      className="text-link-color hover:text-link-color-hover"
-                      title={githubLink}
-                    >
-                      <FontAwesomeIcon icon={faGithub} className="mr-1" />
-                      GitHub repository
-                    </a>
```

Comment:
This section was full of placement issues. I unified. the placements of div, now About and Maintainers look similar and have correct topography.